### PR TITLE
KFSPTS-7140: update with 2017-01-31 financials base code changes

### PR DESCRIPTION
**Pull request by bhutchinson**
This PR should be reviewed and merged after both the nonprod-sql PR for KFSPTS-7140 and the config PR for KFSPTS-7140. 

Also, when this is ready to merge, we need to coordinate with DevOps to make config changes to kfs-dev that correspond to the deployment of the new war built after these changes are merged to the develpo branch.

### CODE REVIEW

**Comment by bhutchinson** [comment]
The unit tests will fail until the related config PR is reviewed and merged.

File: src/main/java/org/kuali/kfs/fp/document/web/struts/CapitalAssetInformationActionBase.java [hunk 1]
```diff
@@ -73,7 +73,7 @@
      * selected asset numbers, the system control amount is redistributed equally among the assets
      * when the distribution method is "distribute cost equally".
      *
-     * @see org.kuali.kfs.kns.web.struts.action.KualiAction#refresh(org.apache.struts.action.ActionMapping,
+     * @see org.kuali.rice.kns.web.struts.action.KualiAction#refresh(org.apache.struts.action.ActionMapping,
```
Comment (db483) [inline src/main/java/org/kuali/kfs/fp/document/web/struts/CapitalAssetInformationActionBase.java]: I wonder why there is a change to the rice version here. It seems that this change was part of the license update, I wonder if it was accidental? I don't have access to the KualiCo financials project so I cannot check out what this code is there.

Comment (bhutchinson) [inline src/main/java/org/kuali/kfs/fp/document/web/struts/CapitalAssetInformationActionBase.java]: This is just a reference in the javadoc comment on a method. An upgrade script I ran changed the package in our overlay file from rice to kfs, but I was changing it back to be the same as what is in the base code.

File: src/main/java/org/kuali/kfs/fp/document/web/struts/CapitalAssetInformationActionBase.java [hunk 2]
```diff
@@ -1066,7 +1061,8 @@ protected void resetCapitalAssetInfo(CapitalAssetInformation capitalAssetInforma
 
     /**
      * Overridden to guarantee that form of copied document is set to whatever the entry mode of the document is
-     * @see org.kuali.kfs.kns.web.struts.action.KualiTransactionalDocumentActionBase#copy(org.apache.struts.action.ActionMapping, org.apache.struts.action.ActionForm, javax.servlet.http.HttpServletRequest, javax.servlet.http.HttpServletResponse)
+     *
+     * @see org.kuali.rice.kns.web.struts.action.KualiTransactionalDocumentActionBase#copy(org.apache.struts.action.ActionMapping, org.apache.struts.action.ActionForm, javax.servlet.http.HttpServletRequest, javax.servlet.http.HttpServletResponse)
```
Comment (bhutchinson) [inline src/main/java/org/kuali/kfs/fp/document/web/struts/CapitalAssetInformationActionBase.java]: same as above

File: src/main/java/org/kuali/kfs/fp/document/web/struts/CapitalAssetInformationActionBase.java (hunk 1 above)
Comment (db483) [inline src/main/java/org/kuali/kfs/fp/document/web/struts/CapitalAssetInformationActionBase.java]: thank you!

**Comment by db483** [comment]
retest this please

**Pull request merged**